sixtyfps_runtime/rendering_backends/gl/graphics_window.rs

```diff
@@ -117,7 +117,7 @@ impl GraphicsWindow {
                 // Make sure that the window's inner size is in sync with the root window item's
                 // width/height.
                 self.set_geometry(s.width, s.height);
-                window_builder.with_inner_size(s)
+                window_builder.with_inner_size(s).with_resizable(false)
```

Comment:
Thanks for the patch! I'm wondering, won't this make even windows that are meant to be resizable not resizable by the user?

Reply:
Indeed I had only tested with slide_puzzle and memory. If preferred-width or preferred-height is set, it also blocks the maximization. I'll fix it as soon as possible.

Reply:
I don't know if this is the best way.